config/service_providers.yml

```diff
@@ -5,12 +5,14 @@ test:
       assertion_consumer_logout_service_url: 'http://localhost:3000/test/saml/decode_slo_request'
       sp_initiated_login_url: 'http://localhost:3000/test/saml'
       cert: 'saml_cert'
+      block_encryption: 'aes256-cbc'
```

Comment:
Also, should we add "encryption: 'none'" so the development hosts below?   Will defaulting to encrypted responses break out current deployments?